src/intern.rs

```diff
@@ -65,6 +65,25 @@ extern {
     ///     [documentation](https://ruby-doc.org/core-2.5.1/doc/extension_rdoc.html#label-Array+Functions)
     pub fn rb_ary_push(array: VALUE, item: VALUE) -> VALUE;
 
+    /// Returns the element at the given index or [`nil`](static.Qnil.html) if none.
+    ///
+    /// * `array` - an [`Array`](static.rb_cArray.html)
+    /// * `idx` - the offset to retrieve. A negative value will count from the end of the array.
+    ///
+    /// # Safety
+    ///
+    /// * Undefined behavior if `array` is not an `Array`
```

Comment:
Probably should call out that this function performs bounds check (which, unlike other items that usually appear in this section, makes things _more_ safe). Nevertheless, I think that's relevant information about safety guarentees.